### ISSUE TITLE
A11y_Playwright.dev_Usable: On Applying High Contrast Aquatic and Desert Mode keyboard focus for the "Search" is not Visible.

### ISSUE THREAD

**Issue by Raghu-addepalli** [microsoft/playwright.dev#1261]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### GitHub Tags:
#A11yUsable;#A11yTCS;#Playwright releases;#P1_Playwright.dev_Web_Jan2024;#A11ySev4;#A11yusablehigh;#ChromiumEdge;#DesktopWeb;#FTP;#HighContrast;#Win11;#P1_WCP

### Environmental Details:
App Name: Playwright.dev
#URL: https://playwright.dev/java/

### OS & Browser Details :
Microsoft Windows 11 Enterprise
Version 22621.3007
Microsoft Edge
Version 120.0.2210.144 (Official build) (64-bit)

### Pre-Requisite:

1. Open settings and Tab to "Accessibility" and press enter.
2. Now Tab to "Contrast Themes" and press enter, "Contrast Theme" page will appear.
3. Now select the themes and refresh the page.

### Repro Steps:

1. Hit the URL: https://playwright.dev/java/
2. Apply High Contrast and TAB to Search Field.
3. Check that in Aquatic and Desert Mode keyboard focus for the "Search" is Visible or not.

### Actual results:
On Applying High Contrast Aquatic and Desert Mode keyboard focus for the "Search" is not Visible.

**Note: Similar issue is also observed in below links.**

https://playwright.dev/docs/test-assertions
https://playwright.dev/docs/running-tests
https://playwright.dev/docs/locators
https://playwright.dev/docs/api/class-locator
https://playwright.dev/docs/writing-tests
https://playwright.dev/docs/api/class-playwright
https://playwright.dev/docs/intro
https://playwright.dev/docs/api/class-page

### Expected results:
On Applying High Contrast Aquatic and Desert Mode keyboard focus should clearly be visible for "Search".

### User Impact:
The visually impaired users will not get where the focus is present if focus is not visible if it not visible in high contrast mode.

### Attachments:

https://github.com/microsoft/playwright.dev/assets/91586743/03be7a11-4e52-4f85-a887-8ea34c4a7503
